src/model/ModelGeometry.i

```diff
@@ -29,6 +29,12 @@
   %ignore openstudio::model::Space::thermalZone;
   %ignore openstudio::model::Space::setThermalZone;
   %ignore openstudio::model::Space::waterUseEquipment;
+
+  %ignore openstudio::model::DaylightingDeviceTubular::transitionZones;
+  %ignore openstudio::model::DaylightingDeviceTubular::addTransitionZone;
+  %ignore openstudio::model::DaylightingDeviceTubular::addTransitionZones;
```

Comment:
C# partial classes, reimplemented later in ModelHVAC.i (where THermalZone/TransitionZone is)